src/PiGui.cpp

```diff
@@ -759,12 +759,11 @@ PiGui::PiGui() :
 	GetFont("pionillium", 14);
 };
 
-const bool PiFace::containsGlyph(unsigned short glyph) const
+const bool PiFace::isValidGlyph(unsigned short glyph) const
 {
 	PROFILE_SCOPED()
-	for (auto range : m_ranges) {
-		if (range.first <= glyph && glyph <= range.second)
-			return true;
+	if(m_invalid_glyphs.find(glyph) == m_invalid_glyphs.end()) {
```

Comment:
Can I recommend using `!m_invalid_glyphs.count(glyph)` here instead? It's shorter, still maintains the same time-complexity, and easily converts to bool.

Reply:
> One thing stands out to me; is there a reason for using `std::unordered_map` over `std::unordered_set`?

Yeah, the reason is my C++ got rusty ;)

> Can I recommend using !m_invalid_glyphs.count(glyph) here instead? It's shorter, still maintains the same time-complexity, and easily converts to bool.

Done!